src/ImageRunModal.scss

```diff
@@ -46,3 +47,46 @@
       margin-block-end: var(--pf-t--global--spacer--xs);
     }
 }
+
+// HACK: It is not possible to mix vertical and horizontal areas within a form. So lets hack it until it is supported
+// upstream by utilizing the upstream code for the FormSection as well.
+// Issue exists since: 2025-08-29. PF v6.3.1
```

Comment:
This is a bit incorrect -- #2207 didn't land yet, so we are still at 6.2.3 at the moment.